packages/components/src/menu/docs/CustomTrigger.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button, Menu, MenuItem, MenuTrigger } from "@hopper-ui/components";
+import { useCallback, useState } from "react";
+
+export default function Example() {
+    const [isOpen, setIsOpen] = useState(false);
+    const handleOpenChange = useCallback((newOpen: boolean) => {
+        setIsOpen(newOpen);
+    }, [setIsOpen]);
+
+    const customTrigger = <Button variant={isOpen ? "primary" : "secondary"}>Custom Trigger</Button>;
```

Comment:
this is not really a custom trigger, there is no difference with the other examples.

We should probably show something like this: https://react-spectrum.adobe.com/react-aria/Menu.html#custom-trigger